rex-engine/src/eval.rs

```diff
@@ -136,12 +139,12 @@ where
     State: Clone + Send + Sync + 'static,
 {
     let var_type = unify::apply_subst(ctx.env.read().await.get(&var.id).unwrap(), &ctx.subst);
-    print!("evaluating var: ({}):({})...", var, var_type);
+    trace!(ctx, "evaluating var: ({}):({})...", var, var_type);
```

Comment:
we can actually just remove these logs, they were very much just "dev logs" not meant to be production-grade.